storybookjs/storybook
# Components: Accordion and AccordionList components

## What I did

Created the new components ```<Accordion />```, ```<AccordionItem />```, ```<AccordionHeader />```, ```<AccordionBody />``` & ```<AccordionList />```

The aim came from a wish to make the list feature used in a11y add-on commonly accessible with a uniform visual expression.

More documentation can be found in the ```official-storybook```

## How to test

Open the ```official-storybook``` example after you spin up with a ```yarn build components```

- Does this need a new example in the kitchen sink apps? Yes, I made stories and documents
- Does this need an update to the documentation? Do not think so

If your answer is yes to any of these, please make sure to include it in your PR.
